src/extension/generalMobilePlatform.ts

```diff
@@ -81,6 +82,35 @@ export class GeneralMobilePlatform {
         return Promise.resolve();
     }
 
+    public modifyOptFromRunArgs(optName: string, value: any): void {
```

Comment:
Why we pass value as any and then check if it is boolean? My opinion is that this function must have two arguments: one for option name and one for arguments list to search(it could be `string` or `string[]`). I don't think that `any` is a good solution here.